src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  task: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ task }) => {
+  const {
+    completed,
+    user,
+    title,
+  } = task;
+
+  return (
+    <>
+      <article
+        className={classNames('TodoInfo',
+          {
+            'TodoInfo--completed': completed,
+          })}
+      >
```

Comment:
A little bit improve readability: 
```suggestion
    className={classNames(
      'TodoInfo',
      { 'TodoInfo--completed': todo.completed },
    )}
```